server/src/KleeRunner.cpp

```diff
@@ -235,6 +235,9 @@ KleeRunner::createKleeParams(const tests::TestMethod &testMethod,
         "--use-tbaa",
         "--output-dir=" + kleeOut.string()
     };
+    if (Paths::isCXXFile(testMethod.sourceFilePath)) {
+        argvData.emplace_back("--libcxx=true");
+    }
```

Comment:
how does libcxx impact the execution? can it be enabled by default?

Reply:
It's try link libcxx bitcode library into module, currently it will be better use it just for C++.